api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -274,16 +289,19 @@ default void setAttribute(AttributeKey<Long> key, int value) {
    * implementation.
    *
    * @param name the {@code Span} name.
+   * @return this.
    */
-  void updateName(String name);
+  Span updateName(String name);
 
   /**
    * Marks the end of {@code Span} execution.
    *
    * <p>Only the timing of the first end call for a given {@code Span} will be recorded, and
    * implementations are free to ignore all further calls.
+   *
+   * @return this.
    */
-  void end();
+  Span end();
```

Comment:
Maybe I'm kind of obsessive - but I don't want to encourage users to do something with a `Span` after `end()` ;)

Reply:
Oh, that's true! Please keep the `void` here, @dengliming